packages/visx-annotation/test/Label.test.tsx

```diff
@@ -9,27 +9,25 @@ describe('<Label />', () => {
     expect(Label).toBeDefined();
   });
   it('should render title Text', () => {
-    expect(
-      shallow(<Label title="title test" resizeObserverPolyfill={ResizeObserver} />)
-        .find(Text)
-        .dive()
-        .text(),
-    ).toBe('title test');
+    shallow(<Label title="title test" resizeObserverPolyfill={ResizeObserver} />)
+      .find('visx-annotationlabel')
+      .forEach(item => {
```

Comment:
can you explain why this needed to be changed? in theory the API of `Text` shouldn't change with these changes right?

a `.forEach` is dangerous because if there are no matches for `visx-annotationlabel`, there would be no assertions and the test would "pass". so if this was needed we should add `expect.hasAssertions()` to the test.

Reply:
That was little embarrassing, cause I'm not familiar with `enzyme`.... I have update `label` test case inspired by `axis`
https://github.com/airbnb/visx/blob/master/packages/visx-axis/test/Axis.test.tsx#L177

Reply:
no worries at all! 😅